Manager/FormManager.php

```diff
@@ -100,18 +102,93 @@ public function save($data, $locale = null, $id = null)
         $translation->setDeactivateCustomerMails(self::getValue($data, 'deactivateCustomerMails'));
         $translation->setChanged(new \DateTime());
 
-        // Add Translation to Form
+        // Add Translation to Form.
         if (!$translation->getId()) {
             $translation->setForm($form);
             $form->addTranslation($translation);
         }
 
-        // Set Default Locale
+        // Set Default Locale.
         if (!$form->getId()) {
             $form->setDefaultLocale($locale);
         }
 
-        // Fields
+        // Fields.
+        $this->updateFields($data, $form, $locale);
+
+        // Receivers.
+        $this->updateReceivers($data, $translation);
+
+        // Save.
```

Comment:
Is there any definition on your site, how a comment has to lool like.

It is MA Rule to add a dot at the end.